### DISCUSSION TITLE
Missing AnimationBehavior Docs?

### DISCUSSION THREAD

**Discussion by s-beji** [CommunityToolkit/Maui#476] [Q&A] [answered]
I would like to implement this behavior in my project. Nevertheless, I tried it without docs and just with a look at the sample project, but I run into a problem. My app crashes caused by this behavior component. So I assumed that just my implementaion could be wrong or incomplete. So I looked in the docs and was surprised that this part is completely missing.

**Comment by VladislavAntonyuk** [comment, marked as answer]
https://github.com/CommunityToolkit/Maui/issues/35
It may be a part of 1.1.0

**Comment by s-beji** [comment]
So it is unusable at the moment?
If this is the case, please share this more clearly.
It is really confusing

**Reply by VladislavAntonyuk** [reply]
it is usable, but not documented

**Reply by bijington** [reply]
@DeepWorksStudios the `AnimationBehavior` implementation did make it in to our GA but we haven't provided any docs - it looks like this slipped through as we documented all finished proposals. The remaining work is currently in progress to get our pre built animations in to use with it.

For now if you need to use it we have this in our sample application:

https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/AnimationBehaviorPage.xaml

**Reply by s-beji** [reply]
Sadly it causes my project to crash as soon i add this feature

**Reply by bijington** [reply]
> Sadly it causes my project to crash as soon i add this feature

I suspect you are using the `EventName` which isn't supported in the current version.

**Reply by s-beji** [reply]
Indeed i was using it any altanativ for using it in combination of an entry (focuse unfocus)

**Reply by bijington** [reply]
No sadly you'll have to wait for the next release. The old implementation didn't play well with `Entry` controls but the new version will fix that

**Comment by CliffAgius** [comment]
Closed as answered

**Comment by s-beji** [comment]
Is it fixed?

**Reply by bijington** [reply]
It should be as far as we are aware. If you find any other issues related to it please open an issue
